js/explore/line_repoActivityTotal.js

```diff
@@ -44,6 +44,23 @@ function draw_line_repoActivityTotal(areaID) {
 		// Christmas
 		var dXmas = getYearDate("12-25",dToday);
 
+		function addDateLine(dateObj,label) {
```

Comment:
Would it be better to just take the date string itself as the argument? Is that `dateObj` used elsewhere?